infra/charts/feast-feature-server/opentelemetry.md

```diff
@@ -0,0 +1,107 @@
+## Adding Monitoring
+To add monitoring to the Feast Feature Server, follow these steps:
+
+### Workflow
+
+Feast instrumentation Using OpenTelemetry and Prometheus - 
```

Comment:
Might be a rookie question, but is both otel and prometheus deployment required in this setup? Won't it work with prometheus server running in feast with just an otel collector?

Reply:
Sure, got that part. I'm asking if `kind: Prometheus` deployment is also strictly required alongside otel components?

Reply:
when the prior PR was closed, we lost a lot of prior discussion / decisions around this topic.
https://github.com/feast-dev/feast/pull/4300#discussion_r1672311834